ui-admin/src/study/participants/enrolleeView/EnrolleeOverview.tsx

```diff
@@ -1,29 +1,72 @@
 import React from 'react'
-import { Enrollee } from 'api/api'
+import Api, { Enrollee, EnrolleeRelation, Profile } from 'api/api'
 import { StudyEnvContextT } from '../../StudyEnvironmentRouter'
 import ParticipantNotesView from './ParticipantNotesView'
 import { dateToDefaultString } from '@juniper/ui-core'
 import KitRequests from '../KitRequests'
+import { InfoCard, InfoCardBody, InfoCardHeader, InfoCardTitle, InfoCardValue } from '../../../components/InfoCard'
+import { useLoadingEffect } from '../../../api/api-utils'
+import LoadingSpinner from '../../../util/LoadingSpinner'
 
 /** Shows minimal identifying information, and then kits and notes */
 export default function EnrolleeOverview({ enrollee, studyEnvContext, onUpdate }:
         {enrollee: Enrollee, studyEnvContext: StudyEnvContextT, onUpdate: () => void}) {
+  const [relations, setRelations] = React.useState<EnrolleeRelation[]>([])
+
+  const { isLoading } = useLoadingEffect(async () => {
```

Comment:
Let's rename in the destructure so it's clear we're not waiting on everything to load
```suggestion
  const { isLoading: isLoadingRelations } = useLoadingEffect(async () => {
```